models/index.js

```diff
@@ -0,0 +1,8 @@
+const fs = require("fs");
+let models = fs.readdirSync(__dirname);
+
+models.forEach( modelFilename => {
```

Comment:
I would recommend to use `models.filter` and then `.map` to export them. But it also will work properly.